src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 15 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
Out-Default Tests.'Out-Default -Transcript' shows up in transcript, but not host
```powershell
Expected exactly 'bye', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Out-Default.Tests.ps1: line 19
19:         & $powershell -noprofile -command $script | Should -BeExactly 'bye'
```
Out-Default Tests.Out-Default reverts transcription state when used more than once in a pipeline
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Out-Default.Tests.ps1: line 24
24:         & $powershell -noprofile -command "Out-Default -Transcript | Out-Default -Transcript; 'Hello'" | Should -BeExactly "Hello"
```
Out-Default Tests.Out-Default reverts transcription state when exception occurs in pipeline
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Out-Default.Tests.ps1: line 28
28:         & $powershell -noprofile -command "try { & { throw } | Out-Default -Transcript } catch {}; 'Hello'" | Should -BeExactly "Hello"
```
Export-Csv.Should append existing properties, add missing properties with empty value, and skip extra properties
```powershell
The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\fab3c552-f888-453a-be06-7b3d41d58505\output.csv' because it is being used by another process.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Export-Csv.Tests.ps1: line 162
162:         $object1 | Export-Csv -Path $testCsv

```